pkg/agent/plugin/workloadattestor/docker/process/winapi.go

```diff
@@ -0,0 +1,252 @@
+//go:build windows
+// +build windows
+
+package process
+
+import (
+	"reflect"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	modkernel32 = windows.NewLazySystemDLL("kernel32.dll")
+	modntdll    = windows.NewLazySystemDLL("ntdll.dll")
+
+	procIsProcessInJob = modkernel32.NewProc("IsProcessInJob")
+
+	procNtQueryObject            = modntdll.NewProc("NtQueryObject")
+	procNtQuerySystemInformation = modntdll.NewProc("NtQuerySystemInformation")
```

Comment:
https://docs.microsoft.com/en-us/windows/win32/api/winternl/nf-winternl-ntquerysysteminformation indicates that [NtQuerySystemInformation may be altered or unavailable in future versions of Windows. Applications should use the alternate functions listed in this topic.]. Is there an alternative to this function?

Reply:
same as before